tests/test_contrib_cern_openid.py

```diff
@@ -127,7 +128,7 @@ def test_account_setup(app, example_cern_openid, models_fixture):
         g.oauth_logged_in_with_remote = ioc.remote_apps["cern_openid"]
 
         login_user(user)
-        assert len(g.identity.provides) == 3
+        assert len(g.identity.provides) == 2
```

Comment:
@zzacharo any idea what change could have caused this? It does not seem related to the changes in this PR.